addons/hr_employee_hourly_wage/models/hr_employee.py

```diff
@@ -7,6 +7,6 @@
 class HrEmployee(models.Model):
     _inherit = 'hr.employee'
 
-    timesheet_cost = fields.Monetary('Cost', currency_field='currency_id',
```

Comment:
Upgrade script required?

Reply:
Yes, forgot to create the pull request for that: https://github.com/odoo/upgrade/pull/3853